README.md

```diff
@@ -143,6 +143,17 @@ It's worth to underline that:
 * Backups for partitions are scheduled randomly within the range of time specified in [config.yaml](./config/config.yaml),
 * It is possible to check the progress via [Task Queues](https://console.cloud.google.com/appengine/taskqueues).
 
+### On-demand table backup
+BBQ provides option for performing on-demand backup for specified table or partition. 
```

Comment:
I'm wondering if we want to document it, as we haven't documented Advanced section at all, i.e. restore from list. We could put some of the docs to the UI tooltips.

Reply:
I agree that UI should inform user almost about everything, but on the other hand, if feature is already described in README, why should we remove it?